docs/static/styles/user-customizations.css

```diff
@@ -468,4 +468,12 @@ kbd {
 
 table {
   table-layout: fixed; /* Keep tables inside the content wrapper */
-}
\ No newline at end of file
+}
+
+/* Styling for variables inside code blocks and such */
+var {
+  color: var(--brand-color-primary-blue);
+  text-decoration: underline var(--brand-color-primary-blue) dotted 2px;
+  font-style: italic;
+  cursor: help;
```

Comment:
😻 Love the display of the help text!!!
Great idea and great implementation!

:nail_care: 
Would making it bold further help to differentiate it from the rest?

💡 Is it possible to show the overlay without delay? (did not check if that's something that can be overridden or not).

💡 Is blue the colour of choice there? Since it's hard to separate it from all the links we already have on docs pages.

Reply:
I think the implementation of the tooltip is from the browser, so not sure about changing the delay. 

With that and the other style questions, I think we'll revisit when we do a more comprehensive redesign. It's not ideal, but probably enough for now.